schedule/__init__.py

```diff
@@ -483,7 +490,8 @@ def run(self):
         """
         logger.info('Running job %s', self)
         ret = self.job_func()
-        self.last_run = datetime.datetime.now()
+        if not isinstance(ret, SkipJob) and ret is not SkipJob:
+            self.last_run = datetime.datetime.now()
```

Comment:
Personally I would invert the if statement like so:

```suggestion
        if isinstance(ret, SkipJob) or ret is SkipJob:
            return ret
        self.last_run = datetime.datetime.now()
```
(untested suggestion)

That would make it more clear that returning `SkipJob` is a special flow. And the normal flow would continue on the same level.